doc/requirements.txt

```diff
@@ -1,6 +1,6 @@
 -r ../pyGHDL/requirements.txt
 
-sphinx>=3.4.2
+sphinx==3.3.1
```

Comment:
Could you add some more information on this bug (like an upstream issue) either to the commit message or as a comment in the file itself?

Reply:
Honestly, I don't know much about it. Lars (from VUnit) found that search was broken with `>=3.3.1` (https://github.com/VUnit/vunit/issues/713). We use the same template in VUnit and here, which is based on sphinx_rtd_theme. Moreover, sphinx_rtd_theme is broken with `docutils>=0.16` (see https://github.com/readthedocs/sphinx_rtd_theme/issues/1115). So, we are temporarily pinning those (see https://github.com/buildthedocs/docker/commit/f7bda0ebb71b51c2b7185cef6b6414efd20ed6cc and https://github.com/buildthedocs/docker/commit/1fd955532d6213bea45c291f44098ea04ac88265) until sphinx_rtd_theme is either fixed or updated (they are expecting to deprecate the current theme and use a new one based on bootstrap, see https://github.com/readthedocs/sphinx_rtd_theme/issues/1086). Then, we will either rebase or update https://github.com/buildthedocs/sphinx.theme for picking the fixes.

Reply:
I see - incidentally, that comment would make a good commit message :)

Reply:
Touché :wink: